app/controllers/exception_hunter/errors_controller.rb

```diff
@@ -2,14 +2,25 @@
 
 module ExceptionHunter
   class ErrorsController < ApplicationController
+    include Pagy::Backend
+
     def index
       @errors = error_groups
     end
 
+    def show
+      Pagy::VARS[:items] = 1
+      @pagy, @error = pagy(last_error)
+    end
```

Comment:
How does it paginate? Doesn't it show the same error again and again?

Reply:
nope, last_error is actually a list, couldn't think of a worse name